packages/client/components/KudosReceivedNotification.tsx

```diff
@@ -43,14 +44,14 @@ const KudosReceivedNotification = (props: Props) => {
     <NotificationTemplate
       message={
         <>
-          {emojiUnicode} {name} gave you kudos in{' '}
+          {emojiUnicode} {name !== '' ? name : 'Anonymous'} gave you kudos in{' '}
```

Comment:
@nickoferrall not sure, if "Someone" for me it feels that if I click on it, I figure out who exactly it was. @acressall What do you think?

Reply:
On the other hand, it may be more obvious in the notification that it is an anonymous user since he has an anonymous avatar. But in the snackbar it may not be obvious, I think

Reply:
Changed to Someone for now

Reply:
I think "someone" is okay. We could also take away the person aspect and just say "You were give kudos in..."

I wonder if we should display a snippet of the kudos, in case there are multiple from the same meeting? If a few people gave you kudos in reflections you might see 4 of the same notification 🤔